collect_app/src/main/res/layout/main_menu.xml

```diff
@@ -54,6 +55,7 @@
                 android:id="@+id/review_data"
                 android:layout_width="0dp"
                 android:layout_height="wrap_content"
+                app:layout_constraintWidth_max="460dp"
```

Comment:
An alternative to declaring the max width on each button would be to declare it for the first one and then constrain each other button (using `constraintStart_toStartOf` and `constraintEnd_toEndOf`) to that. That still means a lot of XML, but does mean you can play with that max width by changing just one attribute.

Reply:
Right, done.